pkg/operator/operator.go

```diff
@@ -260,6 +260,16 @@ func (optr *Operator) maoConfigFromInfrastructure() (*OperatorConfig, error) {
 		return nil, err
 	}
 
+	cloudapiconfig, err := optr.kubeClient.CoreV1().ConfigMaps(optr.namespace).Get("cluster-api-provider-config", metav1.GetOptions{})
```

Comment:
We don't really use 'cluster-api' for object names, we use 'machine-api' in OpenShift.

Reply:
Hi @michaelgugino , Thanks for the review. I corrected it.